.github/workflows/json-schema-upload.yml

```diff
@@ -40,3 +40,10 @@ jobs:
           r2-bucket: ${{ secrets.R2_BUCKET }}
           source-dir: ./packages/json-schema/dist/
           destination-dir: ./config/schema/
+
+      - name: Purge cache for latest schema
```

Comment:
nice!! can we update all the example configs to use latest.json?

Reply:
Updated